x-pack/plugins/inference/common/chat_complete/without_token_count_events.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { filter, OperatorFunction } from 'rxjs';
+import { ChatCompletionEvent, ChatCompletionEventType, ChatCompletionTokenCountEvent } from '.';
+
+export function withoutTokenCountEvents<T extends ChatCompletionEvent>(): OperatorFunction<
```

Comment:
nit: would be nice to have some test coverage for utilites like `without*` and `isSupportConnectorType`.

Reply:
I figured it's so simple it's not worth the overhead, but I'll add some.